### DISCUSSION TITLE
New modding schedule and big developments

### DISCUSSION THREAD

**Discussion by Mrbt0907** [CosmicHikari/Weather2-Remastered#98] [Announcements]
Good afternoon everyone! I want to let you all know that my modding schedule has changed. I am no longer working on weather2 almost every weekday now. I am currently assisting my best friend with his mod, so my time will be split in half for the time being. This week was weather2's week, but next week will be The Titan Mod's week, then back to Weather 2 and so on. Every other week I will be working on this mod essentially. This should not slow down development time much since it allows me to take a longer break from weather2.

However, I have some very big news! I successfully implemented natural storms again and got front objects to work finally. All storms from now on follow the fronts they are associated with. They have basic functionality atm, so don't expect much yet.

I also managed to increase render distance for particles finally. It is now possible to run a minimum of 4 chunk render distance and see storms millions of blocks away. This is configurable along with a few new particle settings to limit how much can render.

![2022-08-25_22 09 24](https://user-images.githubusercontent.com/50820746/190242840-06b76311-f883-4f61-b0a8-d802f3acdc30.png)

There is also now a command that allows you to clear all storm particles on the screen. Very handy for people who don't like clutter all over their screen after you kill a storm.

Particles from the funnel will now increase in size and have a higher velocity depending on how tall layer 0 is set to. You can also now modify the potential size of each stage using a variable found in Config Storm. However, tread carefully as changing it by 0.1 will dramatically increase the size of tornados.

For example, by default it is set to 1.2, which is what you are used to. If you set it to 1.7, all tornados will be almost 1-1 with real life tornados. Below you will see images of clouds that are 1000 blocks high and about 2 miles wide. Really scary stuff. Now, the damage size variable still causes massive lag, so the max damage size was set to 300 blocks in this instance which still gave me 30fps. For bigger tornados, you do still have to increase the max storm size.

![2022-08-25_22 33 09](https://user-images.githubusercontent.com/50820746/190242777-5f133542-4413-472c-8227-b9e616f3708e.png)

![2022-08-26_06 57 31](https://user-images.githubusercontent.com/50820746/190243156-f3943051-26b5-4a7d-ba49-3d52811190e2.png)

I also added in a minimum storm size variable that allows you to scale each storm to a specific size at a minimum. This should also be cool to play with.

There is a lot more features being implemented as we speak, but I will be releasing the new source code once I finish getting things stable and I will release an indev build in my discord server.

This will take some time, but I should have this ready within a month assuming the other mod doesn't consume all of my energy.

Have a great day everyone!

Discord: https://discord.gg/trNQFjZ26w

**Comment by Jaytv25** [comment]
what is the discord link?

**Reply by Mrbt0907** [reply]
https://discord.gg/trNQFjZ26w

**Reply by Justin7654** [reply]
There's a discord??? I never knew that!

**Reply by odvigajlo** [reply]
> https://discord.gg/trNQFjZ26w

Bruh i am banned there for some reason

**Reply by Mrbt0907** [reply]
> > https://discord.gg/trNQFjZ26w
> 
> Bruh i am banned there for some reason

You brough in too many alt accounts. Sorry.

**Reply by odvigajlo** [reply]
> > > https://discord.gg/trNQFjZ26w
> > 
> > 
> > Bruh i am banned there for some reason
> 
> You brough in too many alt accounts. Sorry.

bruh what

**Comment by CosmicHikari** [comment]
Probably not rather just need to be confirmed. Either way, development is
currently on hold for now.

**Comment by CosmicHikari** [comment]
You had 4 or 5 accounts you joined with.

**Reply by odvigajlo** [reply]
> You had 4 or 5 accounts you joined with.
> […](#)
> On Mon, Sep 19, 2022, 04:33 odvigajlo ***@***.***> wrote: https://discord.gg/trNQFjZ26w Bruh i am banned there for some reason You brough in too many alt accounts. Sorry. bruh what — Reply to this email directly, view it on GitHub <[#98 (reply in thread)](https://github.com/Mrbt0907/Weather2-Remastered/discussions/98#discussioncomment-3678641)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AEA2R7Q5QAWSWXM2GAHANQ3V7AQNFANCNFSM6AAAAAAQMWYBEA> . You are receiving this because you commented.Message ID: ***@***.*** com>

WHAT SHOW ME PROOF

**Reply by Mrbt0907** [reply]
I think I know where the confusion is. There is another user in here that registered 3 different accounts with similar names. I'll unban you. Sorry for the inconvenience.

**Reply by odvigajlo** [reply]
> I think I know where the confusion is. There is another user in here that registered 3 different accounts with similar names. I'll unban you. Sorry for the inconvenience.

oh, okay

**Comment by Bagheera68** [comment]
Is this going to come for 1.18.x
(Sry if theres something im not seeing.)

**Reply by Mrbt0907** [reply]
I don't plan to port to 1.18 any time soon.

**Reply by Bagheera68** [reply]
Ok.

**Reply by Bagheera68** [reply]
Keep doing what youre doing; youre doing great! Btw dont feel pressured to do this but updating to 1.18 is a good idea. Everyone would get this mod.

**Reply by Mrbt0907** [reply]
If I get a yes from Coro, I will consider it.

**Reply by Bagheera68** [reply]
Oh ok.

**Comment by oriolprogamer712** [comment]
how will be the version of weather remaster 2.9.0

**Reply by Mrbt0907** [reply]
That is in the future. I am currently working on 2.8

**Comment by oriolprogamer712** [comment]
Some more news about Weather Remaster.

**Reply by Mrbt0907** [reply]
No great progress as of late.

**Reply by CosmicHikari** [reply]
> Some more news about Weather Remaster.

> 1.18 when?

Y'all, keep in mind this is a passion project.

I'm working on a port to 1.16.5 but even so, the entire Minecraft code has changed and each version you literally have to rewrite the mod from scratch. 

If we are pressured to complete a project, it feels more like a job than a passion project. I assure everyone work is being done on the 1.12.2 updates, but please also understand there are times when breaks are necessary for health reasons (we are humans after all!) 🙂

That said, thank you all for checking out the repo and I hope everyone's excited for the future releases!
